onnxruntime/test/providers/provider_test_utils.h

```diff
@@ -378,6 +379,9 @@ class OpTester {
                                 optional<float>(), optional<float>()));
   }
 
+  // Generate the reference outputs with the model file
+  void AddReferenceOutputs(const std::string& model_path);
```

Comment:
@skottmckay , this is the change that uses a model to compute reference outputs, which I meant previously. Please help to review.